docs/documentation/development/bindings/thing-definition.md

```diff
@@ -110,6 +110,22 @@ There exist systemwide channels that are available by default:
 | signal-strength | system.signal-strength | Number       | QualityOfService | Represents signal strength of a device as a Number with values 0, 1, 2, 3 or 4; 0 being worst strength and 4 being best strength.  |
 | low-battery     | system.low-battery     | Switch       | Battery          | Represents a low battery warning with possible values on/off. |
 | battery-level   | system.battery-level   | Number       | Battery          | Represents the battery level as a percentage (0-100%). Bindings for things supporting battery level in a different format (e.g. 4 levels) should convert to a percentage to provide a consistent battery level reading. |
+| power           | system.power           | Switch       | -                | Turn a device on/off. |
+| brightness      | system.brightness      | Dimmer       | Light            | Brightness of a bulb (0-100%). |
+| color           | system.color           | Color        | ColorLight       | Color of a bulb. |
+| color-temperature | system.color-temperature | Dimmer    | ColorLight       | Color temperature of a bulb (0-100%). 0% should be the coldest setting (highest Kelvin value). |
+| location        | system.location        | Location     | -                |Location in lat.,lon.,height coordinates.
+| motion          |system.motion           | Switch       | Motion           | Motion detected by the device (ON if motion is detected).
+| mute            | system.mute            | Switch       | SoundVolume      | Turn on/off the volume of a device. |
+| volume          | system.volume          | Dimmer       | SoundVolume      | Change the sound volume of a device (0-100%). |
+| media-control   | system.media-control   | Player       | MediaControl     | Control for a media player. |
+| media-title     | system.media-title     | String       | -                | Title of a (played) song. |
+| media-artist    | system.media-artist    | String       | -                | Artist of a (played) song. |
+| outdoor-temperature | system:outdoor-temperature | Number: Temperature | Temperature | Current Outdoor Temperature. |
+| wind-direction  | system.wind-direction  | Number:Angle  | Wind            | Wind direction in degrees (0-360°). |
+| wind-speed      | system.wind-speed      | Number:Speed | Wind             | Wind speed |
+| atmospheric-humidity | system:atmospheric-humidity | Number:Dimensionless | Humidity | Atmospheric Humidity in percent. |
+| barometric-pressure | system:barometric-pressure | Number:Pressure | Pressure | Barometric Pressure |
```

Comment:
We use `system:barometric-pressure` (including a `:`) in some cases and `system.wind-speed` (including a `.`) inside. Which one is the right way?

Reply:
Spelling: `Barometric pressure.`